### PR TITLE
[7309] Video controller dialogue does not close after uploading video to S3

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/7309

### CODE REVIEW

File: static-assets/components/cstudio-forms/data-sources/video-S3-upload.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ YAHOO.extend(CStudioForms.Datasources.VideoS3Upload, CStudioForms.CStudioFormDat
 
     var callback = {
       success: function (fileData) {
-        var uri = fileData;
-        var fileExtension = fileData.split('.').pop();
+        var uri = fileData.url ? fileData.url : fileData;
+        var fileExtension = uri.split('.').pop();
```
Comment (rart) [inline static-assets/components/cstudio-forms/data-sources/video-S3-upload.js:61]: What's the reason for the property being one thing or the other? If it is the API, it documented there?

**Review by rart — APPROVED** [review]
.

**Review by rart — CHANGES_REQUESTED** [review]
@jvega190 confirm the second file on the 4.1.x PR is not missing from this PR.

**Comment by jvega190** [comment]
@rart the second file is not missing in this PR, was an issue only happening in 4.1.x (was already fixed in here)

File: static-assets/components/cstudio-forms/data-sources/video-S3-upload.js (hunk 1 above)
Comment (jvega190) [reply static-assets/components/cstudio-forms/data-sources/video-S3-upload.js:61]: According to the API, it's the `fileData.url`, though I got confused at it is that way in the image datasource. Should I update them as part of this PR?

**Review by rart — APPROVED** [review]

**Pull request merged**